classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+class Book < Item
+  attr_reader :publisher, :cover_state
+
+  def initialize(publisher, cover_state)
+    super(publish_date, id)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
```

Comment:
As per UML class diagram, can_be_archived is supposed to be a private method : 
![image](https://user-images.githubusercontent.com/50754458/218752743-171d03f0-2dee-45e0-86a4-ae10f1b64483.png)

I think it would be better if you can make it private 🙏